### ISSUE TITLE
Subscription#withSubscriber may want to (optionally) exclude the Pick

### ISSUE THREAD

**Issue by JPrevost** [MITLibraries/scoap3hub#116]
Now that we have the auto generated Pick Topic, it may be necessary to identify other methods where we really aren't intending to return information about the `Pick` pseudo Topic.

I believe `Subscription#withSubscriber` may be one of those.

Use case: I was trying to determine the total number of Subscriptions a Subscriber has for informational display purposes (which may or may not have value but is used as an example). I tried `@Subscription.withSubscriber(sub.id).size` which seems useful, but if you actually look at what `@Subscription.withSubscriber(sub.id)` returns, it turns out the Pick pseudo topic is included and I'd wager a Subscriber is not interested in that Topic from a "Number of Subscriptions" stand point. (I do think Subs may be interested in all the Items they've Picked though and I may try to expose that in the Dashboard under #101 )

**Comment by richardrodgers** [comment]
Yes, this definitely warrants further thought and work, since in the case you describe, it would be impossible for the caller to know whether to subtract one (since the subscriber maybe never did a manual deposit)

:+1: to investigate/fix
